packages/ListBox/stories/ListBox.keyboard.stories.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Story } from "storybook/assets/styles/common.styles";
+import { getStoryName } from "storybook/storyTree";
+import Toast from "../../Toast/src";
+
+import {
+  SimpleStory,
+  WithDisabled,
+  WithDividers,
+  WithFilterStory,
+  WithInline,
+  WithRawItems,
+} from "./examples/Keyboard";
+
+const storyName = getStoryName("ListBox");
```

Comment:
Can we put this under the `Examples` directory. We just refactored and restructured the stories :).

Reply:
I kinda disagree... this is more like a manual testing story, I'd be tempted to put in in `Backyard/Tests`.